src/cli/common.js

```diff
@@ -60,6 +69,68 @@ export function pluginDirectoryContext(
   };
 }
 
+/**
+ * This method pipelines the data loading needed to run Cred analysis.
+ *
+ * It's a bit of a kitchen sink, but I think it's still valuable to have as its
+ * own method. In particular, there are some non-obvious relationships between the constituent components:
+ * it's important that the identities be computed after loading the dependencies, because
+ * loading the dependencies may create new identities. Thus, I prefer to expose this API which sequences
+ * the steps correctly, rather than trusting the caller to do this themselves.
+ */
+export async function prepareCredData(
+  baseDir: string,
+  config: InstanceConfig
+): Promise<{|
+  graph: WeightedGraph,
+  ledger: Ledger,
+  dependencies: $ReadOnlyArray<DependencyMintPolicy>,
+|}> {
+  const weightedGraph = await loadWeightedGraph(baseDir, config);
+  const ledger = await loadLedger(baseDir);
```

Comment:
Maybe

```javascript
const [weightedGraph, ledger] = await Promise.all([
  loadWeightedGraph(baseDir, config),
  loadLedger(baseDir),
]);
```

? Since we’re steeped in async-land, we might as well try to get some of
the benefits.

Reply:
Done.